tests/EndpointPagesTest.php

```diff
@@ -195,7 +220,18 @@ public function it_assembles_properties_for_a_new_page()
         $this->assertArrayHasKey('date', $dateContent);
         $this->assertCount(1, $dateContent['date']);
         $this->assertArrayHasKey('start', $dateContent['date']);
-        $this->assertEquals($dateValue->format('c'), $dateContent['date']['start']);
+        $this->assertEquals($dateValue->format('Y-m-d'), $dateContent['date']['start']);
+
+        // date (with time)
+        $dateTimeProp = $page->getProperty($dateTimeKey);
+        $this->assertInstanceOf(RichDate::class, $dateTimeProp->getContent());
+        $this->assertFalse($dateTimeProp->isRange());
+        $this->assertEquals($dateValue, $dateTimeProp->getStart());
+        $dateTimeContent = $dateTimeProp->getRawContent();
+        $this->assertArrayHasKey('date', $dateTimeContent);
+        $this->assertCount(1, $dateTimeContent['date']);
+        $this->assertArrayHasKey('start', $dateTimeContent['date']);
+        $this->assertEquals($dateValue->format('c'), $dateTimeContent['date']['start']);
```

Comment:
the getHasTime method from above needs a test to, or doesn't it?

Reply:
yep! Also added filling (when fetching from Notion) of 'hasTime', which we didn't do before. 

✅  Added tests for both cases (when setting a ``Page::class`` and when fetching a Page from Notion)